cclib/parser/turbomoleparser.py

```diff
@@ -983,6 +983,150 @@ def extract(self, inputfile, line):
             
             self.append_attribute("etdips", [tdm_x, tdm_y, tdm_z])
             
+        # Excitation energies with ricc2.
+        #  +================================================================================+
+        #  | sym | multi | state |          CC2 excitation energies       |  %t1   |  %t2   |
+        #  |     |       |       +----------------------------------------+--------+--------+
+        #  |     |       |       |   Hartree    |    eV      |    cm-1    |    %   |    %   |
+        #  +================================================================================+
+        #  | a   |   1   |   1   |    0.3257471 |    8.86403 |  71493.234 |  94.89 |   5.11 |
+        #  | a   |   1   |   2   |    0.3257471 |    8.86403 |  71493.232 |  94.89 |   5.11 |
+        #  | a   |   1   |   3   |    0.3864541 |   10.51595 |  84816.880 |  95.05 |   4.95 |
+        #  | a   |   1   |   4   |    0.3938325 |   10.71673 |  86436.241 |  94.09 |   5.91 |
+        #  | a   |   1   |   5   |    0.3938325 |   10.71673 |  86436.241 |  94.09 |   5.91 |
+        #  | a   |   1   |   6   |    0.4122171 |   11.21700 |  90471.202 |  93.32 |   6.68 |
+        #  | a   |   1   |   7   |    0.4351537 |   11.84114 |  95505.195 |  94.16 |   5.84 |
+        #  | a   |   1   |   8   |    0.4454969 |   12.12259 |  97775.278 |  94.26 |   5.74 |
+        #  | a   |   1   |   9   |    0.4454969 |   12.12259 |  97775.273 |  94.26 |   5.74 |
+        #  | a   |   1   |  10   |    0.5036295 |   13.70446 | 110533.909 |  93.51 |   6.49 |
+        #  +================================================================================+
+        # TODO: Perhaps need a more general way to look for lines like this?
+        if "| sym | multi | state |          CC2 excitation energies       |  %t1   |  %t2   |" in line \
+        or "| sym | multi | state |          ADC(2) excitation energies    |  %t1   |  %t2   |" in line:
+            line = next(inputfile)

```

Comment:
We don't have a written suggestion for this yet, but can you replace these with a `self.skip_lines` call, then a single `line = next(inputfile)`? I've written https://github.com/cclib/cclib/issues/555#issuecomment-882054020 so that we'll add it to docs. You can grep for some examples in the `cclib/parser` folder.

Reply:
Like this?

Reply:
Yes, exactly.

Reply:
Cool got it.

Just out of interest, would the following not also work (replacing both lines)?:
`line = self.skip_lines(inputfile, ["divider", "units", "divider", "value"])[-1]`
Or is this style not prefered?

Reply:
Huh, I guess that could actually work. This might not be a bad idea, there's just no precedent for it. I didn't even remember that `skip_lines` returns the skipped lines. What does everyone else think? @langner @shivupa

Reply:
Yup that does seem like it should work as well. We don't use `skip_lines` like that but I don't see a problem with it. It somewhat feels confusing cause it reads like we are skipping the last line, but then we end up using it. 

> I didn't even remember that skip_lines returns the skipped lines. 

Me neither. I'm curious if the return value is used anywhere.